packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/lib/extract_errors.ts

```diff
@@ -21,34 +28,41 @@ export function extractErrors(
     originalSavedObjectsMap.set(`${savedObject.type}:${savedObject.id}`, savedObject);
   }
   for (const savedObject of savedObjectResults) {
-    if (savedObject.error) {
-      const originalSavedObject = originalSavedObjectsMap.get(
-        `${savedObject.type}:${savedObject.id}`
-      );
-      const title = originalSavedObject?.attributes?.title;
-      const { destinationId } = savedObject;
-      if (savedObject.error.statusCode === 409) {
-        errors.push({
-          id: savedObject.id,
-          type: savedObject.type,
-          meta: { title },
-          error: {
-            type: 'conflict',
-            ...(destinationId && { destinationId }),
-          },
-        });
-        continue;
-      }
+    if (!savedObject.error) {
+      continue;
+    }
+
+    const originalSavedObject = isLegacyUrlAlias(savedObject)
+      ? originalSavedObjectsMap.get(
+          `${savedObject.attributes.targetType}:${savedObject.attributes.sourceId}`
+        ) ??
+        originalSavedObjectsMap.get(
+          `${savedObject.attributes.targetType}:${savedObject.attributes.targetId}`
+        )
+      : originalSavedObjectsMap.get(`${savedObject.type}:${savedObject.id}`);
```

Comment:
__note:__ I'm not yet sure if we should expose legacy URL alias related errors in the API response, maybe we just need to log them.... I cannot find a non-edge use case when we'd get errors for the legacy URL aliases assuming all "preflight" checks we do in the import code. What do you think?